feature-flags/README.md

```diff
@@ -1,21 +1,26 @@
 # Feature Flags
 
 At Buoy we utilize feature flags to enable us to develop features without
-blocking merges into main or releasing often. To accomplish this we typically
-implement one or both of the following:
+blocking merges into main or releasing often.  Feature Flags can fall into one
+of 3 categories:
 
-- [LaunchDarkly] feature flags for client side control or for features which
-  might be per user dependent.
-- Server environment variables for toggling features on the server side.
+1. Release Toggle - This is a feature flag which is used to allow in progress
+   features to be merged into the main branch and go out to production "dark".
+2. Ops Toggles - A feature flag designed to allow us to incrementally release a
+   feature to production, typically targetted at the center level.
+3. Configuration Toggles - These are flags that enable / disable features on a
+   per customer basis and are not considered "temporary".
 
 ## Guide
 
+- Prefer database backed feature flags
```

Comment:
Do we want to mention an exception here for the permanent Configuration Toggles?

Reply:
Good call out. I've updated this to be more specific to release toggles and added an entry for configuration toggles.